core/src/main/java/org/elasticsearch/action/DocWriteResponse.java

```diff
@@ -59,6 +59,7 @@
     private static final String _SEQ_NO = "_seq_no";
     private static final String RESULT = "result";
     private static final String FORCED_REFRESH = "forced_refresh";
+    private static final String _INPUT = "_input";
```

Comment:
Maybe you can try to import ParseField, you can have a look at [ParseField,](https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/common/ParseField.java) and try to change the code.